docs/how-to-guides/adding-a-new-offline-store.md

```diff
@@ -23,16 +23,25 @@ The process for using a custom offline store consists of 4 steps:
 &#x20;OfflineStore class names must end with the OfflineStore suffix!
 {% endhint %}
 
+### Contrib
+
+Generally, new offline stores should go in the contrib folder and should have warnings within the classes that state the offline store is in alpha status. The contrib folder is designated for community contributions that are not fully maintained by Feast maintainers and may contain potential instability and have API changes. It is recommended to add warnings to users that the offline store functionality is still in alpha development and is not fully stable. In order to be classified as fully stable and be moved to the main offline store folder, the offline store should integrate with the full integration test suite in Feast and pass all of the test cases.
```

Comment:
This is generally pretty hard to read imo. Can you make this simpler and more scannable?

e.g. 

### Contrib offline stores
New offline stores go in `sdk/python/feast/infra/offline_stores/contrib/`.

**What is a contrib plugin?**
- Not guaranteed to implement all interface methods
- Not guaranteed to be stable
- Should have warnings for users to indicate this is contrib

**How do I make a contrib plugin an "official" plugin?**
To move an offline store plugin out of contrib, you need:
- GitHub actions (i.e. `make test-python-integration`) is setup to run all tests against the offline store and pass.
- At least two contributors own the plugin (ideally tracked in an OWNERS / CODEOWNERS)